v3/how-to-guides/01-basics/a-pallet-integration/index.mdx

```diff
@@ -15,89 +15,89 @@ You will learn how to include a custom internal pallet that implements an event
 
 1. Create a local pallet called `pallet_something`.
 
-  Import this pallet by adding the following to `/runtime/src/lib.rs`:
+Import this pallet by adding the following to `/runtime/src/lib.rs`:
 
-  ```rust
-  // Import your pallet.
-  pub use pallet_something;
-  ```
+```rust
+// Import your pallet.
+pub use pallet_something;
+```
```

Comment:
Indentation is not correct here, but is correct in all the content that comes before.

Reply:
The indentation below looks incorrect to me